src/main/java/gov/cabinetofice/gapuserservice/dto/UserEmailDto.java

```diff
@@ -0,0 +1,17 @@
+package gov.cabinetofice.gapuserservice.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+
+@Data
+@Builder
+public class UserEmailDto {
```

Comment:
Should we use a `record` here?

Reply:
I don't believe so as we never need to map this back into something we're storing into DB

Reply:
`record` is a good shout, just a cleaner way of defining a DTO. But both solutions will of course work fine